### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**

High level description of the PR.

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-1234](https://federal-spending-transparency.atlassian.net/browse/DEV-1234)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:
- [x] Code review
- [x] Scheduled Demo including Design/Testing/Front-end OR Provided Instructions for Local Testing above and in JIRA (if applicable)
- [x] Tagged Designer in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
- [x] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated (if applicable)
- [x] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged (if applicable)
- [x] Verified cross-browser compatibility
- [x] All componentWillReceiveProps, componentWillMount, and componentWillUpdate in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3339](https://federal-spending-transparency.atlassian.net/browse/DEV-3339)
- [x] Link to this PR in JIRA ticket

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -12,12 +12,16 @@ Technical details for the knowledge of other developers.
...
-- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated (if applicable)
-- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged (if applicable)
+The following are ALL required for the PR to be merged (in ascending LOE):
+- [ ] Link to this PR in JIRA ticket
+- [ ] Tagged Design, Testing, and Code Reviewers in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
+- [ ] Scheduled demo including Design/Testing/Front-end `if applicable`
+- [ ] Provided instructions for testing in JIRA (for benefit of testing) and PR (for benefit of code reviewer) `if applicable`
 - [ ] Verified cross-browser compatibility
+- [ ] Verified mobile/tablet/desktop/monitor responsiveness
+- [ ] Added Unit Tests for new Container Components `if applicable`
```
Comment (ebdabbs) [inline pull_request_template.md]: I would either remove this (as it is expected as a prerequisite to creating the PR), or if we choose to keep it as a reminder, add that we should also update unit tests for helper functions and reducers.

File: pull_request_template.md [hunk 2]
```diff
@@ -12,12 +12,16 @@ Technical details for the knowledge of other developers.
...
-- [ ] Tagged Designer in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
-- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated (if applicable)
-- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged (if applicable)
+The following are ALL required for the PR to be merged (in ascending LOE):
+- [ ] Link to this PR in JIRA ticket
+- [ ] Tagged Design, Testing, and Code Reviewers in `#us-ux-frontend` Slack Channel in the thread under the Post from Github for their SA
+- [ ] Scheduled demo including Design/Testing/Front-end `if applicable`
+- [ ] Provided instructions for testing in JIRA (for benefit of testing) and PR (for benefit of code reviewer) `if applicable`
 - [ ] Verified cross-browser compatibility
+- [ ] Verified mobile/tablet/desktop/monitor responsiveness
```
Comment (ebdabbs) [inline pull_request_template.md]: 👍

File: pull_request_template.md (hunk 1 above)
Comment (jonhill13) [reply pull_request_template.md]: yeah, this should be removed. or in addition to the above it should say added tests for any methods or additional logic in containers.

File: pull_request_template.md [hunk 3]
```diff
@@ -12,12 +12,16 @@ Technical details for the knowledge of other developers.
...
+- [ ] Added Unit Tests for methods in Container Components, reducers, and helper functions `if applicable`
```
Comment (jonhill13) [inline pull_request_template.md]: Comma after Container

**Pull request merged**